SPECS/kernel/kernel-uki.spec

```diff
@@ -13,7 +13,9 @@
 # We need to temporarily turn it off by disabling xsaves until the problem
 # is fixed on Azure. Since shadow stack depends on xsaves, disabling xsaves
 # ensures the feature bit for shadow stack is also turned off.
-%define cmdline console=ttyS0 noxsaves
+
+#%define cmdline console=tty0 console=ttyS0=115200 audit=0 rd.shell=1 rd.break=pre-pivot noxsaves
```

Comment:
clean these arguments up. Do we want to have the toolkit inform these in anyway? Is that even possible?